src/sonic-yang-models/yang-models/sonic-port-ingress-profile-list.yang

```diff
@@ -0,0 +1,55 @@
+module sonic-port-ingress-profile-list {
+    namespace "http://github.com/Azure/sonic-port-ingress-profile-list";
+    prefix bpg;
+
+    import sonic-extension {
+        prefix sonic-ext;
+    }
+
+    import sonic-port {
+        prefix prt;
+    }
+
+    import sonic-buffer-profile {
+        prefix bpf;
+    }
+
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description
+        "SONIC PORT INGRESS PROFILE LIST";
+
+    revision 2021-07-01 {
+        description
+            "Initial revision.";
+    }
+
+
+    container sonic-port-ingress-profile-list {
+
+        container PORT_INGRESS_PROFILE_LIST {
```

Comment:
Where do we process this table in the backend?

Reply:
Renamed it to BUFFER_PORT_INGRESS_PROFILE_LIST, it is processed in bufferorch.cpp in the backend.